libs/net/services/Config/ServiceOptions.cs

```diff
@@ -25,10 +25,20 @@ public class ServiceOptions
     public int RetryDelayMS { get; set; } = 5000;
 
     /// <summary>
-    /// get/set - Default millisecond delay between process cycle.  This stops run-away threads (default: 30 second).
+    /// get/set - Default millisecond delay between process cycle.  This stops run-away threads (default: 30 seconds).
     /// </summary>
     public int DefaultDelayMS { get; set; } = 30000;
 
+    /// <summary>
+    /// get/set - Whether to restart a service after a critical sequence of failures.
+    /// </summary>
+    public bool AutoRestartAfterFailure { get; set; } = true;
```

Comment:
These config options should have been implemented a long time ago when we added this functionality to a few of the services.